test/async-resolve-from-view-engine.js

```diff
@@ -0,0 +1,649 @@
+/*global describe:false, it:false, before:false, beforeEach:false, after:false, afterEach:false*/
+'use strict';
+
+var fs = require('fs'),
+    path = require('path'),
+    express = require('express'),
+    engine = require('../index'),
+    assert = require('chai').assert,
+    assertions = require('./assertions');
+    var munger = require('engine-munger');
```

Comment:
`munger` is not used. is this the file where a test of the `view.lookup` method would go?